.all-contributorsrc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "files": ["CONTRIBUTORS.md"],
+  "imageSize": 150,
+  "commit": false,
+  "contributors": [
+    {
+      "login": "feng-tao",
+      "name": "Tao Feng",
```

Comment:
will every contributor has an entry here?

Reply:
Yep, but we need to add them manually by calling the bot on PRs